pnpm-lock.yaml

```diff
@@ -6184,7 +6188,11 @@ importers:
       rimraf: ^4.4.0
       typescript: ~4.5.5
     dependencies:
+      '@fluid-internal/stochastic-test-utils': link:../../test/stochastic-test-utils
```

Comment:
THis looks like its left over from testing with linking?

Reply:
I just this just means its a workspace dep, so this is what we want.

Reply:
yeah, that's correct. common-utils is in its own workspace so it doesn't look the same.